user-doc/annotation.md

```diff
@@ -401,12 +402,53 @@ the alternative one has modified the representation of each entity in some way.
 
 Supported JSON payload patterns:
 
-- `{` ... _context_ `:` [ _sname_, _tname_] `,` ... `}`: The table identified by _sname_:_tname_ is an alternative table to be used instead of the annoted table in the specified context. 
+- `{` ... _context_ `:` [ _sname_, _tname_] `,` ... `}`: The table identified by _sname_:_tname_ is an alternative table to be used instead of the annoted table in the specified context.
 
 A alternative table or view which abstracts another table _SHOULD_ have a non-null (psuedo) primary key which is also a foreign key to the base storage table. The base storage table is the one bearing this annotation. Otherwise, a consuming application would not know how to navigate from one abstracted representation of an entity to another representation from the base storage tables.
- 
+
 See [Context Names](#context-names) section for the list of supported _context_ names. It is assumed that any application context that is performing mutation (record creation, deletion, or editing) MUST use a base entity storage table that is not an abstraction over another table. However, the use of the `detailed` or `compact` context MAY offer an abstraction that augments the presentation of an existing record. An application offering mutation options while displaying an existing entity record might then present the data from the `detailed` or `compact` abstraction but only offer editing or data-entry controls on the fields available from the base storage table.
 
+### 2016 Asset
+
+`tag:isrd.isi.edu,2016:asset`
+
+This key indicates that the annotated table may be interpreted as asset metadata.
+
+Supported JSON payload patterns:
+
+- `null` or `{}`: Default heuristics apply.
+- `{`... `"internal":` [_column_, ...] ...`}`: The one or more named _columns_ store internal identifiers for the asset, used for efficient normalized storage in the database but not meaningful to typical users. The referenced columns MUST each comprise a single-column key for the table.
+- `{`... `"identifier": ` _column_ ...`}`: The _column_ stores a resolvable, globally unique, identifier for the asset. In its simplest form this MAY be satisfied by supplying a URL to the resource. The identifier MUST be _globally unique_ which is satisfiable by any fully qualified HTTP URL, for example. In any case the identifier MUST be _resolvable_ meaning that a client can retrieve the identified asset either directly or indirectly through a resolution service. More formal identifier schemes may also be considered such as MINID, DOI, ARK, or PURL. It is up to client tooling to recognize and resolve identifiers in such schemes.
```

Comment:
@mikedarcy not sure what you mean.  A current ERMrest response can have relative URLs (up to the DBA and users what they put in tables) and it should be evaluated relative to the ERMrest URL that was used to get the response.

In practice, we have used absolute paths, e.g. 
- Request is `GET /ermrest/catalog/1/entity/Foo` against `https://www.example.com`
- Therefore `https://www.example.com/ermrest/catalog/1/entity/Foo` is the URL to use as context when interpreting the response
- Response has a URL column containing values like `/hatrac/foo/bar.txt`
- The response values, when interpreted in the request context, mean something like `https://www.example.com/hatrac/foo/bar.txt` etc.